src/audio/volume/volume_hifi3_with_peakvol.c

```diff
@@ -21,12 +21,16 @@ LOG_MODULE_DECLARE(volume_hifi3, CONFIG_SOF_LOG_LEVEL);
 
 #include "volume.h"
 
-#if SOF_USE_HIFI(3, VOLUME)
+// Hifi4 is disabled, see bug https://github.com/thesofproject/sof/issues/9213
+// Hifi5 is not there yet.
+#if SOF_USE_HIFI(3, VOLUME) || SOF_USE_HIFI(4, VOLUME) || SOF_USE_HIFI(5, VOLUME)
 
 #if CONFIG_COMP_PEAK_VOL
 
 #include <xtensa/tie/xt_hifi3.h>
 
+#warning VOLUME 3 is on
```

Comment:
should not spoil, in the volume HIFi component is not available comments and logs, so only this we can use

Reply:
Don't put this in production. This was just sample debug code I posted in a comment to help. We're trying to always keep the build warning-free by default.

Otherwise LGTM. Other reviewers should consider all the other alternatives I listed above, the have subtly different trade-offs.